catalog/ui/catalog-ui-search/package.json

```diff
@@ -5,9 +5,10 @@
   "version": "0.0.1",
   "license": "LGPL-3.0",
   "scripts": {
-    "start": "grunt start",
+    "start": "grunt start --webpackConfig=./webpack/config/dev.js",
```

Comment:
This is a dev only npm script.  The normal build will still use the prod version.